foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -56,6 +56,10 @@ An important implication to notice here is that when you specify `flex-grow` or
 
 `flex-basis` simply sets the initial size of a flex item, so any sort of `flex-grow`ing or `flex-shrink`ing starts from that baseline size. The shorthand value defaults to `flex-basis: 0%`. The reason we had to change it to `auto` in the `flex-shrink` example is that with the basis set to `0`, those items would ignore the item's width, and everything would shrink evenly. Using `auto` as a flex-basis tells the item to check for a width declaration (`width: 250px`).
 
+### flex-auto
+
+`flex: auto` is one of the short hands of flex. When `flex: auto` is called it inherits the values of `flex-grow: 1 flex-shrink: 1 flex:basis: auto` or `flex: 1 1 auto` using the flex shorthand. Note that `flex: auto` is not the default value when using the flex short hand despite the name being "auto" which may be slightly confusing at first. The default value for flex is `flex: 0 1 auto` or `flex: initial` which we won't get too in depth about here.
```

Comment:
**issue (typo):** `flex:basis: auto` -> `flex-basis: auto`

**nitpick:** Some of the language here could be tweaked just a bit. "Called" sort of implies a method/function so perhaps we should use something like "specified". I'm also not sure that "inherits" is the right word choice here either. 

Maybe something like: "When `flex: auto` is specified it is equivalent to using the `flex-grow: 1` `flex-shrink: 1` and `flex-basis: auto` properties or using the flex shorthand `flex: 1 1 auto`."

Reply:
Hey! My bad for seeing this so late, I don't get the github notifications for some odd reason. Let me know your thoughts on the wording within the recent push and hopefully I cleared the confusion there, I  have these drafts that I made that we can come back too if it isn't as clear as we want it to be yet


- When `flex: auto` is defined it is equivalent to the values of `flex-grow: 1 flex-shrink: 1 flex-basis: auto (your option)
- When `flex: auto` is specified it is equivalent to the values of `flex-grow: 1 flex-shrink: 1 flex-basis: auto` or `flex: 1 1 auto
- When `flex: auto` is defined as a flex parameter it is equivalent to the values of `flex-grow: 1 flex-shrink: 1 flex-basis: auto